### PR TITLE
Dịch "lr-scheduler_vn" - Phần 2

### LINKED ISSUE #1978: Dịch "lr-scheduler_vn" - Phần 2
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_optimization/lr-scheduler_vn.md#L49-L173

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by davidnvq**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Điền số issue sau dấu `#` (không có dấu cách): Close #1978

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn. Bạn đã kiểm tra rằng mỗi đoạn văn đã nằm một dòng?

* [x] Điền tên đầy đủ của bạn vào mục **Những người thực hiện** ở cuối tập tin.

* [ ] Điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Lưu ý cho người phản biện (Reviewer)

1. Chỉ review nếu Pull Request (PR) này đã sẵn sàng cho việc review. Một PR sẵn sàng khi PR đó không có label `status: wip`.
2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

### CODE REVIEW

File: chapter_optimization/lr-scheduler_vn.md [hunk 1]
```diff
@@ -138,7 +142,9 @@ Note how the training accuracy keeps on increasing while progress in terms of te
 The gap between both curves indicates overfitting.
 -->
 
-*dịch đoạn phía trên*
+Ta hãy xem điều gì sẽ xảy ra khi ta gọi thuật toán với các thiết lập mặc định, chẳng hạn tốc độ học bằng $0.3$ và huấn luyện với $30$ bước lặp.
```
Comment (cuongvng) [inline chapter_optimization/lr-scheduler_vn.md:145]:
```suggestion
Ta hãy xem điều gì sẽ xảy ra khi ta gọi thuật toán với các thiết lập mặc định, chẳng hạn tốc độ học bằng $0.3$ và huấn luyện với $30$ epoch.
```

File: chapter_optimization/lr-scheduler_vn.md [hunk 2]
```diff
@@ -138,7 +142,9 @@ Note how the training accuracy keeps on increasing while progress in terms of te
...
+Lưu ý rằng độ chính xác trên tập huấn luyện vẫn tiếp tục tăng trong khi độ chính xác trên tập kiểm tra không tăng thêm khi tới một giá trị nào đó.
```
Comment (cuongvng) [inline chapter_optimization/lr-scheduler_vn.md:146]:
```suggestion
Lưu ý rằng độ chính xác trên tập huấn luyện vẫn tiếp tục tăng trong khi độ chính xác trên tập kiểm tra không tăng thêm khi đạt giá trị nào đó.
```

File: chapter_optimization/lr-scheduler_vn.md [hunk 3]
```diff
@@ -153,15 +159,17 @@ train(net, train_iter, test_iter, num_epochs, loss, trainer, ctx)
...
 
 <!--
 One way of adjusting the learning rate is to set it explicitly at each step.
 This is conveniently achieved by the `set_learning_rate` method.
 We could adjust it downward after every epoch (or even after every minibatch), e.g., in a dynamic manner in response to how optimization is progressing.
 -->
 
-*dịch đoạn phía trên*
+Một cách để điều chỉnh tốc độ học là thiết lập giá trị của tốc độ học tường minh ở mỗi bước lặp.
+Điều này có thể đạt được bằng hàm `set_learning_rate`.
```
Comment (cuongvng) [inline chapter_optimization/lr-scheduler_vn.md:171]:
```suggestion
Điều này có thể đạt được bằng phương thức `set_learning_rate`.
```

File: chapter_optimization/lr-scheduler_vn.md [hunk 4]
```diff
@@ -153,15 +159,17 @@ train(net, train_iter, test_iter, num_epochs, loss, trainer, ctx)
...
+Ta có thể  hạ giá trị tốc độ học xuống sau mỗi epoch (hay thậm chí sau mỗi minibatch) như là một cách phản hồi khi quá trình tối ưu đang diễn ra.
```
Comment (cuongvng) [inline chapter_optimization/lr-scheduler_vn.md:172]:
```suggestion
Ta có thể hạ giá trị tốc độ học xuống sau mỗi epoch (hay thậm chí sau mỗi minibatch) như là một cách phản hồi khi quá trình tối ưu đang diễn ra.
```

File: chapter_optimization/lr-scheduler_vn.md [hunk 5]
```diff
@@ -80,7 +80,11 @@ Since most of the code is standard we just introduce the basics without further
 See :numref:`chap_cnn` for a refresher as needed.
 -->
 
-*dịch đoạn phía trên*
+Hãy bắt đầu với một ví dụ đơn giản với ít chi phí tính toán nhưng đủ để minh họa một vài điểm cốt lõi.
+Ta sử dụng LeNet cải tiến (thay thế hàm kích hoạt `sigmoid` bằng `relu` và hàm gộp trung bình bằng hàm gộp cực đại) và áp dụng trên tập dữ liệu Fashion-MNIST.
+Hơn nữa, để có hiệu năng tốt, ta lai hoá mạng.
```
Comment (rootonchair) [inline chapter_optimization/lr-scheduler_vn.md:85]:
```suggestion
Hơn nữa, để có hiệu năng tốt, ta hybrid hoá mạng.
```

Mọi người từng thảo luận về từ hybrid và kết quả cuối cùng là... giữ nguyên :))

File: chapter_optimization/lr-scheduler_vn.md [hunk 6]
```diff
@@ -153,15 +159,17 @@ train(net, train_iter, test_iter, num_epochs, loss, trainer, ctx)
...
+## Bộ Định thời
 
 <!--
 One way of adjusting the learning rate is to set it explicitly at each step.
 This is conveniently achieved by the `set_learning_rate` method.
 We could adjust it downward after every epoch (or even after every minibatch), e.g., in a dynamic manner in response to how optimization is progressing.
 -->
 
-*dịch đoạn phía trên*
+Một cách để điều chỉnh tốc độ học là thiết lập giá trị của tốc độ học tường minh ở mỗi bước lặp.
```
Comment (rootonchair) [inline chapter_optimization/lr-scheduler_vn.md:170]:
```suggestion
Một cách để điều chỉnh tốc độ học là thiết lập giá trị của tốc độ học một cách tường minh ở mỗi bước lặp.
```
Sorry for picky...
